img/template_test.go

```diff
@@ -1,37 +1,30 @@
 package img
 
 import (
+	"fmt"
 	"image"
 	"image/gif"
 	"testing"
 
-	"github.com/stretchr/testify/assert"
+	"github.com/flopp/go-findfont"
+	"github.com/stretchr/testify/suite"
 )
 
-var testGetGif = []struct {
-	path string
-	err  bool
-}{
-	{"fixtures/earth.gif", false},
-	{"fixtures/non-existent.gif", true},
-	{"fixtures/badfile.gif", true},
+type TemplateTestSuite struct {
```

Comment:
This is basically the same structure we have in ImageTestSuite. Adding a todo to unify the base struct in a comment might be a good idea.

Reply:
Perfectly agree with you. I deliberately duplicate them as the first refactoring step, waiting for a clearer strategy.
Comment added.